src/DssSpell.sol

```diff
@@ -58,74 +60,145 @@ contract DssSpellAction is DssAction {
     //
     // uint256 internal constant X_PCT_RATE = ;
 
-    // ---------- Math ----------
-    uint256 internal constant BILLION = 10 ** 9;
-
-    // ---------- Addesses ----------
-    address internal immutable MCD_PAUSE = DssExecLib.getChangelogAddress("MCD_PAUSE");
-    address internal immutable LINE_MOM  = DssExecLib.getChangelogAddress("LINE_MOM");
-
+    // ---------- Contract addresses ----------
+    GemAbstract internal immutable MKR = GemAbstract(DssExecLib.mkr());
+
+    InputConduitJarLike internal immutable MCD_PSM_PAX_A_INPUT_CONDUIT_JAR = InputConduitJarLike(DssExecLib.getChangelogAddress("MCD_PSM_PAX_A_INPUT_CONDUIT_JAR"));
+    JarLike internal immutable MCD_PSM_PAX_A_JAR                           = JarLike(DssExecLib.getChangelogAddress("MCD_PSM_PAX_A_JAR"));
```

Comment:
```suggestion
    address internal immutable MCD_PSM_PAX_A_INPUT_CONDUIT_JAR = InputConduitJarLike(DssExecLib.getChangelogAddress("MCD_PSM_PAX_A_INPUT_CONDUIT_JAR"));
    address internal immutable MCD_PSM_PAX_A_JAR               = JarLike(DssExecLib.getChangelogAddress("MCD_PSM_PAX_A_JAR"));
```

Reply:
- https://github.com/makerdao/spells-mainnet/blob/c198914645a86d51cee32d35749e8561caa14996/archive/2024-02-07-DssSpell/DssSpell.sol#L78-L79
- https://github.com/makerdao/spells-mainnet/blob/c198914645a86d51cee32d35749e8561caa14996/archive/2024-02-22-DssSpell/DssSpell.sol#L75-L76

I checked it on the reviewer checklist but these has been used like this previously. 
I do not have strong opinion on it but I'm not sure what does [except where the archive pattern differs from this pattern](https://github.com/makerdao/pe-checklists/blob/45b65c3829e66713e84af2ce89c2209df6e5c4be/spell/spell-reviewer-mainnet-checklist.md?plain=1#L311) would mean then.